### ISSUE TITLE
Pulmonary pseudocavity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6707]
**Preferred term label:**
Pulmonary pseudocavity

**Synonyms**

**Definition (free text, please give PubMed ID)**

A pseudocavity appears as an oval or round area of low attenuation in lung nodules, masses, or areas of consolidation that represent spared parenchyma, normal or ectatic bronchi, or focal emphysema rather than cavitation. These pseudocavities usually measure less than 1 cm in diameter. They have been described in patients with adenocarcinoma, bronchioloalveolar carcinoma, and benign conditions such as infectious pneumonia.

PMID:18195376

Comment
See Figure 57 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
